tests/docker-compose.yml

```diff
@@ -4,4 +4,9 @@ services:
   elasticsearch:
     image: elasticsearch:2-alpine
     ports:
-    - "9200"
\ No newline at end of file
+    - "9200"
+
+  wiki_es:
+    image: elasticsearch:2-alpine
+    ports:
+        - "9201:9200"
```

Comment:
you shouldn't expose the port as it will makes it impossible to run more than one test instance

Reply:
done